javascripts/components/dinoForm.js

```diff
@@ -0,0 +1,45 @@
+
+import { infoModal } from "./infoModal.js";
+import { addDinos, singleDino } from './addDinos.js'
+import { kennel, dinos } from './../../assets/data/dinoData.js'
+const clickAdd = () =>{
+    $('#submitForm').on('click', (e)=>{
+    e.preventDefault()
+     addDinoForm()
+     }
+    )
+}
+
+const randomID = () => {
+    const random = [Math.floor(Math.random() * 100)];
+    return random;
+  };
+
+const addDinoForm = () => {
+    const name = $("#name").val();
+    const type = $("#type").val();
+    const age = $("#age").val();
+    const owner = $("#owner").val();
+  
+    const imageUrl = $("#imageUrl").val();
+  
+    kennel.push({
+      id: randomID(),
+      name: name,
+      type: type,
```

Comment:
For these names of properties, try drying your code a bit using this: https://alligator.io/js/object-property-shorthand-es6/